### PR TITLE
Personalization: Fix for beta dropdown not rendering after login refactor

### PR DESCRIPTION
**Pull request by ncksllvn**
Resolves department-of-veterans-affairs/vets.gov-team/issues/10105

### CODE REVIEW

**Comment by BBarone1** [comment]
LGTM

**Review by BBarone1 — COMMENTED** [review]
LGTM - if only I had write access :cry:

**Review by hpjaj — APPROVED** [review]
LGTM

**Pull request merged**
